dotnet-trace/test.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+set -x
+
+PROJNAME=dotnettrace
+FILENAME=dotnettrace.nettrace
+SPEEDSCOPENAME=dotnettrace.speedscope.json
+REPORTNAME=dotnettrace.nettrace.etlx
+
+dotnet tool update -g dotnet-trace > /dev/null
```

Comment:
I think redirecting output to /dev/null means we lose command and debugging output. What do you think about leaving that in? That way, when something fails, we can at least see what the output of the command was? And even when things pass, we can see, for example, what version of the tool was installed.

Reply:
Sounds good to me, I'll change it now.